content/sponsors/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Sponsors"
+layoutBackgroundBlur: false
+---
+
+London Gophers is a volunteer-run organisation that seeks to remain as friendly and welcoming as possible to everyone. 
+Therefore, membership of London Gophers is 100% free, and we do not charge for attending events. Not only that, 
+but we provide food and drinks at all of our events. We therefore rely on sponsorship to cover our costs. 
+Thank you to our sponsors!
+
+## Venue Sponsors
+These are the venues that have hosted us in the past few months. Thank you!
+
+### TrueLayer
+Instant access to open banking -
+Easily integrate next-generation payments and financial data
+into any app. Build powerful products your customers love.
+{{< figure
+src="images/2022_12_truelayer.png"
+alt="TrueLayer Logo"
+link="https://truelayer.com/"
+target="_blank"
+height="70%"
+width="70%"
+default=true
+>}}
+
+### SaltPay
+We exist to serve local businesses -
+Fast and secure payment solutions that can help small and medium-sized businesses to better manage and grow their operations.
+{{< figure
+src="images/2022_11_saltpay.png"
+alt="SaltPay Logo"
+link="https://www.saltpay.co/"
+target="_blank"
+height="70%"
+width="70%"
+default=true
+>}}
```

Comment:
We might need to add an white background to this image tags (at least based on what I can see on the preview)

Reply:
I wonder if we can create a custom CSS class for that